test/onnx/test_pytorch_onnx_caffe2.py

```diff
@@ -1159,7 +1159,7 @@ def __init__(self):
                 super(MyModel, self).__init__()
 
             def forward(self, x, y):
-                return x.repeat(y.size()[0] / 2, y.size()[1] * 2)
+                return x.repeat(y.size()[0] // 2, y.size()[1] * 2)
```

Comment:
is this some strange ONNX treat-a-size as a Tensor thing?

Reply:
Moved to follow-up PR.